docs/DeveloperGuide.md

```diff
@@ -65,6 +65,36 @@ The Parser component:
 We pass in the`itemList`to the`Command`object instead of using a global variable to ease testing. This way, we can 
 add, delete and update`Expenditure`entries in a new`itemList`during testing without affecting the actual`itemList`.
 
+### Command component 
+The source can be found in [command](https://github.com/AY2122S2-CS2113T-T10-4/tp/blob/master/src/main/java/seedu/mindmymoney/command)
+
+![command_diagram](images/command_diagram.png)
+<br/> Fig 5 - Command Diagram
+
+The Command component consists of `Command` abstract class, `AddCommand`, `ByeCommand`, `DeleteCommand`, `HelpCommand`
+, `ListCommand`and `UpdateCommand` class.
+
+The Command component:
+- `Command` class is an abstract class and `AddCommand`, `ByeCommand`, `DeleteCommand`, `HelpCommand`, `ListCommand`
+and `UpdateCommand`extends the `Command` class.
```

Comment:
Maybe can move these 2 lines before "The Command component:"

Reply:
fixed